scss/components/table.scss

```diff
@@ -188,25 +202,46 @@ $block: #{$fd-namespace}-table;
     &--fixed-wrapper {
         position: relative;
     }
+
     &--fixed {
         overflow-x: scroll;
         padding-left: 200px;
 
-        td {
+        .#{$block}__cell {
             min-width: 200px;
         }
     }
 
     &--no-borders {
+        @include fd-reset();
         border: none;
-        & tbody tr {
+
+        .#{$block}__body .#{$block}__row {
             border: none;
         }
     }
 
     &--striped {
-        tbody tr:nth-child(even) {
+        .#{$block}__body .#{$block}__row:nth-child(even) {
             @include fd-var-color("background-color", fd-color-state("hover"), --fd-color-background-hover);
         }
     }
+
+    &.#{$fd-namespace}-has-first-child-text-align-right {
+        .#{$block}__cell:first-child {
+          text-align: right !important;
+        }
+    }
+    &.#{$fd-namespace}-has-last-child-text-align-right {
+        .#{$block}__cell:last-child {
+            text-align: right !important;
+        }
+    }
+    @each $i in 2,3,4,5,6,7,8,9 {
+        &.#{$fd-namespace}-has-nth-child-#{$i}-text-align-right {
+            .#{$block}__cell:nth-child(#{$i}) {
+            text-align: right !important;
+            }
+        }
+    }
```

Comment:
Do these class names make anyone else cringe? 🤢 Don't need to change them now, but there _has_ to be a better way.